api/views.py

```diff
@@ -1481,7 +1481,10 @@ def user_save_view(request):
     personal_code = ud.get("personal_code", 0)
     rmis_resource_id = ud["rmis_resource_id"].strip() or None
     snils = ud.get("snils").strip() or ''
-    cabinet = ud.get("cabinet").strip() or ''
+    try:
+        cabinet = ud.get("cabinet").strip()
```

Comment:
Решение не лучшее. 
Правильнее будет
```suggestion
        cabinet = (ud.get("cabinet") or "").strip()
```

Я так понимаю проблема в том, что там может быть None.
Тогда решение, которое я написал сработает

Reply:
Try except не нужны тогда